libs/base/Data/Nat.idr

```diff
@@ -45,6 +48,17 @@ isItSucc : (n : Nat) -> Dec (IsSucc n)
 isItSucc Z = No absurd
 isItSucc (S n) = Yes ItIsSucc
 
+||| A hystorical synonym for `IsSucc`
+public export
+0 NonZero : Nat -> Type
+NonZero = IsSucc
+
+-- Remove as soon as 0.8.0 (or greater) is released
+public export %inline
+%deprecate -- use ItIsSucc instead
```

Comment:
If you put the suggestion to use ItIsSucc instead into a code doc then it should show up in the warning for deprecation which is a nice touch.

Reply:
Sounds reasonable, done